tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckWidths.java

```diff
@@ -103,7 +119,8 @@ public Limit(double d, double e, Measure displayWidth, LimitType maximum, Specia
             this(d, e, displayWidth, maximum, placeholders, false);
         }
 
-        boolean hasProblem(String value, List<CheckStatus> result, CheckCLDR cause, Boolean aliasedAndComprenehsive) {
+        boolean hasProblem(String path, String value, List<CheckStatus> result, CheckCLDR cause, Boolean aliasedAndComprenehsive) {
+            double factor = 1d;
```

Comment:
"aliasedAndComprenehsive" is misspelled, but that's not new and it must be used consistently or the compiler would have complained

Reply:
fixed